### PR TITLE
add multi_rake

### PR DESCRIPTION
**Pull request by rluria14**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rluria14** [comment]
@oblute @benhuff agree to maintain?

File: recipes/multi-rake/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - cld2-cffi >=0.1.4
+    - numpy >=1.14.4
+    - pip
+    - pyrsistent >=0.14.2
+    - python >=3.5
+    - regex >=2018.6.6
```
Comment (scopatz) [inline recipes/multi-rake/meta.yaml]: Do you really need most of these at install time?

Comment (rluria14) [reply recipes/multi-rake/meta.yaml]: @scopatz thanks, host reqs have been updated

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**